src/test/java/org/assertj/core/presentation/StandardRepresentation_toStringOf_AtomiceReferences_Test.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2021 the original author or authors.
+ */
+package org.assertj.core.presentation;
+
+import org.junit.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.*;
+import java.util.concurrent.atomic.AtomicMarkableReference;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.concurrent.atomic.AtomicStampedReference;
+
+public class StandardRepresentation_toStringOf_AtomiceReferences_Test {
+  static Representation standardRepresentation = new StandardRepresentation();
```

Comment:
use `StandardRepresentation.STANDARD_REPRESENTATION` instead, no need for a field, use statically imported `STANDARD_REPRESENTATION` directly